x-pack/test/functional/page_objects/search_navigation.ts

```diff
@@ -35,12 +37,13 @@ export function SearchNavigationProvider({ getService, getPageObjects }: FtrProv
         }
       });
     },
-    async navigateToIndexDetailPage(indexName: string) {
-      await retry.tryForTime(60 * 1000, async () => {
-        await common.navigateToApp(`elasticsearch/indices/index_details/${indexName}`, {
-          shouldLoginIfPrompted: false,
-        });
+
+    async navigateToIndexDetailPage() {
+      await solutionNavigation.sidenav.expectLinkExists({ text: 'Index Management' });
+      await solutionNavigation.sidenav.clickLink({
+        deepLinkId: 'management:index_management',
       });
+      await searchIndexDetailsPage.openIndicesDetailFromIndexManagementIndicesListTable(0);
```

Comment:
Not sure about selecting first index is the expected behavior for this function
I mean when you call it by name `navigateToIndexDetailPage` it hard to figure which index will be selected. But this is debatable

Reply:
Yeah i did think this through. We could confirm if selected index is same as index by checking browser url has intended indexName. But thinking if its necessary since our  test cases should work for any search index and we do index clean up before we start tests.   

I am working on alternate approach to test this.

Reply:
changed to find intended indexName from list of index names and click on indexName from table - 0972fc71a00bc377e941872cca8e4a5eb99b6b98